src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int BOUND = 100;
+
+    public Figure getRandomFigure() {
+        String randomColor = new ColorSupplier().getRandomColor().toLowerCase();
+        Random random = new Random();
+        int index = random.nextInt(FiguresList.values().length);
+        String randomFigure = FiguresList.values()[index].toString();
```

Comment:
better use an instance of your enum and replace strings in cases with constants from enum
```suggestion
        String randomFigure = FiguresList.values()[index];
```